oshc/main/models.py

```diff
@@ -2,5 +2,28 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.forms import ModelForm, Textarea
 
 # Create your models here.
+
+
+class chatSession(models.Model):
+    title = models.CharField(max_length=128, help_text="Require title of the session")
```

Comment:
`help_text="Title of the session"` would be more appropriate. All the fields are mandatory by default. You don't need to add an explicit `require` :)

The same goes for all the fields in the model.